bin/deploy-core

```diff
@@ -42,7 +42,9 @@ seth send "$MCD_DEPLOY" "deployTaxationAndAuctions(address)" "$MCD_GOV"
 seth send "$MCD_DEPLOY" "deployLiquidator()"
 
 # Deploy End
-seth send "$MCD_DEPLOY" "deployEnd()"
+MCD_ESM_SUN=${MCD_ESM_SUN:-"0x0"}
```

Comment:
So `0x0` fails with `seth`, it should be `0x0000000000000000000000000000000000000000`